tests/unit/test_tinystories.py

```diff
@@ -0,0 +1,25 @@
+import pytest
+from transformers import AutoConfig
+
+MODEL_NAMES = [
+    "roneneldan/TinyStories-Instruct-1M",
+    "roneneldan/TinyStories-Instruct-3M",
+    "roneneldan/TinyStories-Instruct-8M",
+    "roneneldan/TinyStories-Instruct-28M",
+    "roneneldan/TinyStories-Instruct-33M",
+    "roneneldan/TinyStories-1Layer-21M",
+    "roneneldan/TinyStories-2Layers-33M",
+    "roneneldan/TinyStories-Instuct-1Layer-21M",
+    "roneneldan/TinyStories-Instruct-2Layers-33M",
+]
+
+
+def test_all_models_exist():
```

Comment:
```suggestion
def test_all_tinystories_models_exist():
```

Please move this test to `test_hooked_transformer`